### PR TITLE
ghsl-urban-delineation

### PR DESCRIPTION
**Pull request by eorlitova**
ghsl-urban-delineation folder  and ghsl-urban-delineation.yaml  first draft

### CODE REVIEW

**Comment by Schpidi** [comment]
thanks @eorlitova, this looks great to me.

@Dorothyrono are you the right person to ask to merge this? thanks

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/ghsl-urban-delineation.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        Code: 2
+      - Name: DUC_DenseUrbanCluster_EPSG4326_RefYear.shp
+        Description: Shapefile containing delineated polygons of Dense Urban Clusters in selected AOI and reference year. Polygons are projected to WGS84 coordinate reference system.
+        Code: 23
+      - Name: SDUC_SemiDenseUrbanCluster_EPSG4326_RefYear.shp
+        Description: Shapefile containing delineated polygons of Semi-Dense Urban Clusters in selected AOI and reference year. Polygons are projected to WGS84 coordinate reference system.
+        Code: 22
+      - Name: PUC_PeriUrbanCluster_EPSG4326_RefYear.shp
+        Description: Shapefile containing delineated polygons of Peri-Urban Clusters in selected AOI and reference year. Polygons are projected to WGS84 coordinate reference system.
+        Code: 21
```
[the remark this answers is not in the document]
Comment (Dorothyrono) [reply collections/ghsl-urban-delineation.yaml:61]: @eorlitova 
A suggestion that could work for now, is to introduce a space in the names 
e.g if you add a space just before EPSG...   `HDC_UrbanCentre_ EPSG4326_RefYear.shp` it breaks the name and the table is displayed in much better  way

image

**Comment by eorlitova** [comment]
@Dorothyrono thanks for your comments.  I have made changes according to them.

**Comment by eorlitova** [comment]
@Dorothyrono could you please let me know if you can now merge the  ghsl-urban-delineation to the public-collections.
Thanks. Erika

**Review by Dorothyrono — APPROVED** [review]

**Comment by Dorothyrono** [comment]
Merging

**Pull request merged**